xbmc/settings/Settings.h

```diff
@@ -195,6 +195,7 @@ class CSettings : public CSettingsBase, public CSettingCreator, public CSettingC
   static const std::string SETTING_PVRCLIENT_MENUHOOK;
   static const std::string SETTING_PVRTIMERS_HIDEDISABLEDTIMERS;
   static const std::string SETTING_MUSICLIBRARY_SHOWCOMPILATIONARTISTS;
+  static const std::string SETTING_MUSICLIBRARY_USEARTISTSORTNAME;
```

Comment:
No need to declare it here, it is only used in one file so move it to that file. (see https://github.com/xbmc/xbmc/pull/13856#discussion_r186278587)

Reply:
Currently the value is only used in one file yes, but if sorting is moved to the DB (more efficient than the roll-our-own sorting in fileitemlist) then other files may want it.  It is a global setting, not just a UI one.

This also surprizes me as I thought the idea was to have all the items that the user sees on the screens from the main Settings menu defined in one place. Unlike the other local declarations we have so far where the setting is seen on a specifc dialog, the useartistsortname option is seen on Settings>Media>Music main screen. That is how I interpreted what we were doing, so I would like some clarififcation before I make it local to GUIViewStateMusic.cpp